Tasks/AzurePowerShellV5/azurepowershell.ts

```diff
@@ -160,6 +160,33 @@ async function run() {
     catch (err) {
         tl.setResult(tl.TaskResult.Failed, err.message || 'run() failed');
     }
+    finally {
+        let RemoveAzContextPath = path.join(path.resolve(__dirname), 'RemoveAzContext.ps1');
+        let removeScripts = `${RemoveAzContextPath} -ErrorAction continue`
```

Comment:
Is there any intention to use this variable? If not please remove it. Also note that with your current solution `RemoveAzContext.ps1` runs without `-ErrorAction continue`. Please considen incorporating it.